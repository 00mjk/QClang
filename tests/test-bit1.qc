int main() {
    bit a;
    return 0;
}
