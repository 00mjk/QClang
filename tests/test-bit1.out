OPENQASM 2.0;
include "qelib1.inc";
qreg q[1];
creg c[1];
h q;
creg n_a_b0[1];
